actionview/test/template/form_collections_helper_test.rb

```diff
@@ -39,6 +39,13 @@ def with_collection_check_boxes(*args, &block)
     assert_select "label[for=user_active_no]", "No"
   end
 
+  test "collection radio allows non-English values for labels" do
+    with_collection_radio_buttons :user, :title, ['господин', 'госпожа'], :to_s, :to_s
+
+    assert_select "label[for=user_title_господин]", "господин"
+    assert_select "label[for=user_title_госпожа]", "госпожа"
+  end
```

Comment:
@lugray I'm only adding a test for `collection_radio_buttons` because that's where I faced the issue and I don't see any other use-case where it could be important. (`sanitized_value` is used only by `collection_check_boxes` and `collection_radio_buttons`)